cobc/flag.def

```diff
@@ -93,11 +93,12 @@ CB_FLAG_NQ (1, "callfh", 9,
 	  "                        as external provided EXTFH interface module"))
 
 CB_FLAG_RQ (cb_ebcdic_table, 1, "ebcdic-table", 0, 14, /* cf cconv.h for all available tables */
-	_("  -febcdic-table=[DEFAULT|RESTRICTED-GC|IBM|GCOS]\tdefine EBCDIC translation table:\n"
+	_("  -febcdic-table=[DEFAULT|RESTRICTED-GC|IBM|GCOS|<filename>]\tdefine EBCDIC translation table:\n"
 	  "                        * default: translation to extended ASCII as per MF\n"
```

Comment:
I haven't catched that before - either the values should be in all upper-case as in the parameter, or lower-case there (I _think_ the former is the way to go). Please adjust.